universal-login-commons/lib/core/utils/messages/computeGasData.ts

```diff
@@ -3,18 +3,27 @@ import {isProperHexString} from '../hexStrings';
 
 export const ZERO_BYTE_GAS_COST = 4;
 export const NON_ZERO_BYTE_GAS_COST = 68;
+export const NEW_NON_ZERO_BYTE_GAS_COST = 16;
+export const GAS_FIXED = '50000';
```

Comment:
Move to constants/gas

Reply:
I moved it in the incoming PR. But a good point! 👍